#patrol name
string name
---
float32 angle
